nimare/meta/mkda.py

```diff
@@ -305,19 +250,17 @@ def _run_fwe_permutation(self, params):
         between vFWE and cFWE.
         """
         iter_ijk, iter_df, conn, voxel_thresh = params
-        iter_ijk = np.squeeze(iter_ijk)
-        iter_df[["i", "j", "k"]] = iter_ijk
```

Comment:
I can't find where you fill in the permutation's default DataFrame with its IJK values.

Reply:
If they aren't inserted somewhere, then the "random" permutations would just be done on the original dataset.

Reply:
Compare the old version (first image) to the new version (second image):

![image](https://user-images.githubusercontent.com/8228902/98726580-b2e53680-2364-11eb-8826-db0d16c2387a.png)

![image](https://user-images.githubusercontent.com/8228902/98726597-b7a9ea80-2364-11eb-90f1-48f20c3013be.png)

Reply:
Yikes, good catch! This is pretty bizarre though, because the results look basically the same coming out of `ALE` and `MKDADensity`. Will look into it.

Reply:
Oh, nevermind—I was thinking of the uncorrected results. I haven't actually looked at what comes out of the corrector. Could well be totally broken.